src/lib.rs

```diff
@@ -55,19 +54,17 @@
 
     ## The `scad!` macro
     The most important part of the crate is the `scad!` macro. The first parameter
-    of the macro is the element type of the object we want to create which should be 
+    of the macro is the element type of the object we want to create which should be
     an instance of the `ScadElement` enum. If you only want to create a single scad
     object, you can simply end the macro invocation after the parent like this:
 
 
     ```
-    # #[macro_use]
-    # extern crate scad;
-    # use scad::*;
+    use scad::*;
 
-    # fn main(){
-        scad!(Cube(vec3(10., 10., 10.)));
-    # }
+    fn main(){
+      scad!(Cube(vec3(10., 10., 10.)));
+    }
```

Comment:
I think the # es should be here, IIRC they hide the main part in the docs which are fairly pointless

Reply:
Yes, in fact the main function can be left out completely